index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/dcd34e45ea.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <title>Portfolio</title>
+</head>
+<body>
+    <section class="flex-container outer-container">
+        <header class="flex-container">
+            <nav class="flex-container nav-bar">
+                <!--- Item to display logo -->
+                <div class="flex-item" id="logo">
+                    Welcome
+                </div>
+                <div class="flex-item remove-nav-item">
+                    <a href="#">Portfolio</a>
+                </div>
+                <div class="flex-item remove-nav-item">
+                    <a href="#">About</a>
+                </div>
+                <div class="flex-item remove-nav-item">
+                    <a href="#">Contact</a>
+                </div>
+                <div class="flex-item" id="link">
+                    <i class="fa-solid fa-bars"></i></i>
+                </div>
```

Comment:
- [OPTIONAL] It is the best practice, for semantics reason, and the right position for convention and user experience for your hamburger to be at the right position and wrap in a `nav` tag on mobile devices. As suggested by the previous reviewer, kindly wrap it in a `nav` tag.

Reply:
@Qoosim. My brother, maximum of the changes you devised are OPTIONAL. Only one minor adjustment is required and for that you are consuming my extra review. If it is near to the template, then please let it go because we have just started working on it.